api/search/index.js

```diff
@@ -1,7 +1,7 @@
 const os = require('@opensearch-project/opensearch')
 
 const options = process.env.NODE_ENV === 'development'
-  ? { node: 'http://localhost:9200' }
+  ? { node: 'http://opensearch:9200' }
```

Comment:
Can you figure out a way to not need to hard code this?

This will break local dev for anyone not running in docker

Reply:
I've now moved it to .env, which should cover all bases assuming non-docker dev setups keep it blank (or set it to localhost) in their .env.

Reply:
Woot